solutions/bronze/usaco-1301.mdx

```diff
@@ -78,8 +76,6 @@ public class WatchingMooloo {
 		br.close();
 		for (int i = 0; i < n; i++) { days[i] = Long.parseLong(daysInput[i]); }
 
-		Arrays.sort(days);
```

Comment:
wait why was this removed again

Reply:
the elements are already in sorted order as per stated by the question